client/js/shout.js

```diff
@@ -382,6 +382,19 @@ $(function() {
 		}
 	}
 
+	if ($.cookie("highlights") !== null) {
+		$("#highlights").val($.cookie("highlights")).change();
+	}
+
+	var highlights = [];
+
+	$("#highlights").on("change", function() {
+		var self = $(this);
+		var string = self.val();
+		highlights = string.split(",");
```

Comment:
I would probably run [`trim()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim) on the result in case people enter things like `highlight1, highlight2` (and it's worth adding a comment to explain this too):

```js
> "highlight1, highlight2,highlight3".split(",").map(function (h) { return h.trim(); });
[ 'highlight1', 'highlight2', 'highlight3' ]
```